src/components/organisms/UiMenu/_internal/UiMenuItem.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <UiListItem
+    class="ui-menu-item"
+    v-bind="listItemAttrs"
+  >
+    <UiButton
+      :class="['ui-button--outlined', 'ui-menu-item__button']"
```

Comment:
```suggestion
      class="ui-button--outlined ui-menu-item__button"
```
R: use :class="[]" when you want to pass static and computed classes e.g.
```
:class="['ui-button--outlined ui-menu-item__button', {'button--is-disabled': isButtonDisabled}]"
```

Reply:
but here we need to translate `ui-menu-item--is-selected` to `ui-button--is-selected`
```
  <UiButton
    :class="['ui-button--outlined ui-menu-item__button', {'ui-button--is-selected': isSelected}]"
  >
```